src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -60,8 +60,11 @@ Env::AppBundle::AppBundle(Application* app_)
 Env::AppBundle::AppBundle(std::unique_ptr<Config const> config)
 {
     auto logs = std::make_unique<Logs>();
-    owned = make_Application(
-        std::move(config), std::move(logs));
+    auto timeKeeper_ =
+        std::make_unique<ManualTimeKeeper>();
+    timeKeeper = timeKeeper_.get();
```

Comment:
@vinniefalco So we have to be careful not to let `owned` get destroyed without clearing or updating `timeKeeper`, yes?